src/components/App.jsx

```diff
@@ -28,36 +29,28 @@ class ContactsApp extends React.Component {
   }
 
   render() {
-    const { t } = this.props
+    const { t, deleteContact } = this.props
 
     return (
       <Layout monocolumn="true">
         <Main>
           {flag('switcher') && <FlagSwitcher />}
-          <ContactsSelectionBar trashAction={this.props.deleteContact} />
+          <ContactsSelectionBar trashAction={deleteContact} />
           <Query query={query}>
-            {({ data: contacts, fetchStatus }) => {
-              if (fetchStatus === 'loading') {
-                return (
-                  <>
-                    <Content>
-                      <SpinnerContact
-                        size="xxlarge"
-                        loadingType="fetching_contacts"
-                      />
-                    </Content>
-                  </>
-                )
-              } else {
-                return (
-                  <>
-                    {contacts.length >= 1 && <Header right={<Toolbar />} />}
-                    <Content>
-                      <ContactsList contacts={contacts} />
-                    </Content>
-                  </>
-                )
-              }
+            {({ data: contacts, fetchStatus, hasMore, fetchMore }) => {
+              return (
+                <>
```

Comment:
I don't know this! Is this syntactic sugar for `React.Fragment`?

Reply:
Yep!